lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -329,6 +329,7 @@ public static function fileUploadFunction(array $params, $template)
         }
 
         $form = isset($params['form']) ? $params['form'] : null;
+        $imageResizeOption = isset($params['imageResizeOption']) && empty($params['imageResizeOption']) ? false : true;
```

Comment:
Czy my wszystkich parametrów funkcji smarty nie robimy w formacie `nazwa_parametru`?

Reply:
Różnie. Przykład: `$params['customOnChange']` i `$params['location_name']`. To którą wersję użyć?

Reply:
częście chyba używamy `location_name`.

Reply:
no i po prostu 'image_resize`. Słowo `option` jest nadmiarowe, bo to wszystko są opcje/parametry ;-)

Reply:
Zrobione. Zdecydowanie bez `option` bo to miało tylko jakiś sens w pierwotnej wersji ;-)